mod/src/main/java/zsawyer/mods/mumblelink/mumble/UpdateData.java

```diff
@@ -200,4 +217,27 @@ protected String generateContext(Minecraft game, int maxLength) {
     private Vec3 getTopVec(Minecraft game) {
         return game.player.getUpVector(1f);
     }
+
+
+    /**
+     * Make people in other dimensions far away so that they're muted.
+     * <p>
+     * reimplementation of https://github.com/magneticflux-/fabric-mumblelink-mod/blob/12727324ae9ecfc9c6b0ab5b604e824d43cfffa1/src/main/kotlin/com/skaggsm/mumblelinkmod/client/ClientMumbleLinkMod.kt#L136
+     *
+     * @param game             the source to get live data from
+     * @param originalPosition the original position to be offset
+     */
+    public static void applyDimensionalOffset(Minecraft game, float[] originalPosition) {
+        ResourceKey<Level> dimension = game.player.level.dimension();
+        if (dimension == null) {
+            // silently ignoring because it would become too spammy
+            return;
+        }
+
+        int configuredOffset = MumbleLinkImpl.dimensionalHeight();
+        int hash = LinkAPIHelper.stableHash(dimension.toString());

```

Comment:
I am not sure if this dimension.toString is the same as what fabrics implementation uses.